DependencyInjection/Configuration.php

```diff
@@ -72,8 +71,15 @@ public function getConfigTreeBuilder()
                         ->scalarNode('stream_factory')->defaultNull()->end()
                     ->end()
                 ->end()
-            ->end()
-        ;
+                ->arrayNode('toolbar')
+                    ->addDefaultsIfNotSet()
+                    ->info('Extend the debug profiler with inforation about requests. This could only be active when the kernel is in debug mode.')
+                    ->children()
+                        ->enumNode('enabled')->values([true, false, 'auto'])->defaultValue('auto')->end()
```

Comment:
and add an info() here saying that by default, the toolbar is active when kernel.debug is true, but you can force it to on or off with this option.